### PR TITLE
feat(logger): first implementation

### PR DESCRIPTION
**Pull request by CptSchnitz**

### CODE REVIEW

File: tests/index.spec.ts [hunk 1]
```diff
@@ -0,0 +1,10 @@
+import jsLogger from '../src';
+
+describe('plus', function () {
```
Comment (syncush) [inline tests/index.spec.ts:3]:
```typescript
describe('jsLogger', function () {
```

File: package.json [hunk 2]
```diff
@@ -52,12 +50,17 @@
...
+    "jest-html-reporters": "^2.1.3",
     "prettier": "^2.2.1",
     "pretty-quick": "^3.1.0",
     "standard-version": "^9.0.0",
     "ts-jest": "^26.4.4",
     "ts-node": "^9.1.1",
     "typescript": "^3.8.3"
+  },
+  "dependencies": {
+    "@types/pino": "^6.3.6",
```
Comment (syncush) [inline package.json:62]: @types/pino should be a devDep

**Review by syncush — CHANGES_REQUESTED** [review]

File: tests/index.spec.ts (hunk 1 above)
Comment (CptSchnitz) [reply tests/index.spec.ts]: done

File: package.json (hunk 2 above)
Comment (CptSchnitz) [reply package.json]: done

**Review by syncush — APPROVED** [review]

**Pull request merged**